src/main/java/it/gov/pagopa/pu/debtpositions/util/Utilities.java

```diff
@@ -58,4 +62,14 @@ public static String getRandomIUD() {
   public static String getRandomicUUID() {
     return UUID.randomUUID().toString().replace("-", "");
   }
+
+  public static String generateRandomIupd(String orgFiscalCode) {
+    String uuid = UUID.randomUUID().toString();
+    String lastUuidPart = uuid.substring(uuid.length() - 12);
```

Comment:
uuid has a fixed length, instead of calculate the difference, lets set the final value